mantidproject/mantid
# Update EQSANS q resolution

Small change to use an effective pixel size to compute the Q resolution for EQSANS.
This should not affect any other instrument using TOFSANSResolution.

**To test:**
- Review the code
- Make sure the tests run.

Fixes #16652.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
